api/inc/svc_exports.h

```diff
@@ -116,6 +116,7 @@
 /* SVC immediate values for hardcoded table (call from unprivileged) */
 #define UVISOR_SVC_ID_UNVIC_OUT        UVISOR_SVC_FIXED_TABLE(0, 0)
 #define UVISOR_SVC_ID_REGISTER_GATEWAY UVISOR_SVC_FIXED_TABLE(3, 0)
+#define UVISOR_SVC_ID_BOX_INIT_NEXT    UVISOR_SVC_FIXED_TABLE(5, 0)
```

Comment:
I would like to add a comment in the code about the previously used slots to avoid any confusion in the future. What was 1 used for? What was 2 used for (different from 1)?